src/components/Logo.jsx

```diff
@@ -1,14 +1,26 @@
+import { useLocation } from "react-router-dom";
+
 import Button from "./UI/Button";
+import PropTypes from "prop-types";
+
+const Logo = ({ styles, type, destination }) => {
+  const location = useLocation();
+  const pathName = location.pathname;
 
-const Logo = () => {
   return (
     <Button
-      destination={"/"}
-      styles="text-36 font-bold bg-gradient-to-r from-[#9996EF] to-[#F9C7D4] text-transparent bg-clip-text"
+      destination={destination}
+      styles={`font-bold bg-gradient-to-r from-[#9996EF] to-[#F9C7D4] text-transparent bg-clip-text ${styles}`}
     >
-      Bloblow
+      {pathName === "/" && type === "nav" ? "Welcome" : "Bloblow"}
```

Comment:
```suggestion
      {pathName === "/" ? "Welcome" : "Bloblow"}
```

Logo의 prop으로 받는 `type`의 역할이 따로 있을까요? 위 코드로 적용해보면, 결과적으로 기능에 차이가 없더라구요. `pathName`만으로 충분하지 않을까 판단되는데, `type`을 추후 활용할 여지가 있을까요? :)

Reply:
type은 로고의 위치를 나타냅니다. "nav" bar에 위치하는 것은 nav 타입인데요, 홈페이지 중앙에 저희 서비스 이름이 뜨는데도 불구하고 "nav" bar에 동일하게 "Bloblow"가 떠서 위와 같은 코드를 작성했어요.